lib/ecto_crux.ex

```diff
@@ -269,6 +269,29 @@ defmodule EctoCrux do
       ######################################################################################
       # READ MULTI
 
+      @doc """
+      [Repo] Fetches all results using the query.
+          query = from b in Baguette, where :kind in ["tradition"]
+          best_baguettes = Baguettes.find_by(query)
+      """
+      def unquote(:find_by)(%Ecto.Query{} = query) do
```

Comment:
you move them here because:
`unquote(:find_by)(%Ecto.Query{} = query, opts)`
match before 
`unquote(:find_by)(%Ecto.Query{} = query, opts)`
?

if so, thx ! (I will add some unit tests ...)

but you you forgot one L324

Reply:
Yes, because I had these kind of bugs: 
```
iex(9)> Relation.Schema.Audiences.find_by(Relation.Schema.Audiences.init_query())
** (Protocol.UndefinedError) protocol Enumerable not implemented for #Ecto.Query<from a0 in Relation.Schema.Audience> of type Ecto.Query (a struct). This protocol is implemented for the following type(s): Ecto.Adapters.SQL.Stream, Postgrex.Stream, DBConnection.PrepareStream, DBConnection.Stream, Date.Range, File.Stream, Function, GenEvent.Stream, HashDict, HashSet, IO.Stream, List, Map, MapSet, Range, Stream
    (elixir 1.10.2) lib/enum.ex:1: Enumerable.impl_for!/1
    (elixir 1.10.2) lib/enum.ex:141: Enumerable.reduce/3
    (elixir 1.10.2) lib/enum.ex:3383: Enum.map/2
    (relation 0.1.0) lib/relation/schema/audiences.ex:4: Relation.Schema.Audiences.find_by/1
```

I let the one L324, since the following function above is using it : https://github.com/delight-data/ecto_crux/pull/14/files/e837a7bb11309651db00cefd182173cd759fb333#diff-eecfd784ef011f6964237c4ad560ff57678a90941c58bd585e2dec0ebd81a332R318-R322

But you are right, it might not change anything to regroup all of them, I'm modifying it !